metadata.rb

```diff
@@ -4,7 +4,7 @@
 license 'Apache 2.0'
 description 'Installs and configures the collectd monitoring daemon.'
 long_description 'Installs and configures the collectd monitoring daemon.'
-version '2.2.2'
+version '2.2.3'
```

Comment:
I think we should probably bump this to 2.3 since it changes how the system is managing the actual daemon. Is there any consideration for systems that were using an older cookbook and now start to use the new one?